charts/topolvm/values.yaml

```diff
@@ -0,0 +1,308 @@
+image:
+  # image.repository -- TopoLVM image repository to use.
+  repository: quay.io/topolvm/topolvm-with-sidecar
```

Comment:
Some user wants to use lightweight container that has no sidecar binary (see  #196).
Could you add settings to use light topolvm image and official sidecar images?

Reply:
supported: https://github.com/topolvm/topolvm/pull/302/commits/c36339067fbedc86bb9bdf8c816d8ad2275252c3